components/ad/smartbanner/src/index.scss

```diff
@@ -51,11 +51,15 @@
     }
   }
 
-  &-visible {
+  &--visible {
     margin-top: 70px;
   }
 
-  &-hidden {
+  &--hidden {
     margin-top: 0;
   }
```

Comment:
Now I know 😅 
Turns out this modifier is appended to the App Container DIV:
`<div id="App" class="sui-AdSmartbanner-visible">`